spec/system/provider_interface/provider_manages_application_interviews_spec.rb

```diff
@@ -156,12 +156,24 @@ def and_an_interview_has_been_created(date)
 
   alias_method :and_another_interview_has_been_created, :and_an_interview_has_been_created
 
-  def when_the_first_interview_has_happened
+  def when_the_first_interview_has_only_just_happened
     Timecop.travel(2020, 3, 2, 13, 0, 0)
     provider_signs_in_using_dfe_sign_in
     visit provider_interface_application_choice_interviews_path(application_choice)
   end
 
+  def then_the_completed_interview_still_shows_as_upcoming
+    expect(page).to have_css('.app-interviews > :nth-child(2)', text: 'Upcoming interviews')
+    expect(page).to have_css('.app-interviews > :nth-child(3)', text: '2 March 2020')
+    expect(page).to have_css('.app-interviews > :nth-child(4)', text: '3 March 2020')
+  end
+
+  def when_the_first_interview_happened_yesterday
+    Timecop.travel(2020, 3, 3, 9, 0, 0)
+    provider_signs_in_using_dfe_sign_in
+    visit provider_interface_application_choice_interviews_path(application_choice)
+  end
+
```

Comment:
Is it worth moving the interview view logic in a component and verifying that the correct forms are rendered at that level than attempting to expand the system spec so it caters for all possible scenarios.